WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp

```diff
@@ -1248,16 +1268,14 @@
 
         WebUtils.VM.taskDetails.displayDate = ko.pureComputed(function(){
             return WebUtils.VM.taskDetails.date();
-            /*var dateString = WebUtils.VM.taskDetails.date();
-
-            if (dateString){
-                return (moment(dateString, "MM/DD/YYYY").calendar(null, {
-                    sameElse: 'MMM D[,] YYYY'
-                }).split(" at"))[0]
+        });
 
-            }else {
-                return " ";
-            }*/
+        WebUtils.VM.taskDetails.mlsPerKgCal = ko.pureComputed(function(){
+            if (WebUtils.VM.taskDetails.mlsPerKg() >= 20){
```

Comment:
Instead of the if statement this could be:

return !(WebUtils.VM.taskDetails.mlsPerKg() >= 20)

Reply:
made the recommended change